### ISSUE TITLE
arcdps+radial+imgui+selectrenderer(gw2hook)

### ISSUE THREAD

**Issue by ludwigma** [gw2-addon-loader/GW2-Addon-Manager#149]
crashes on start with this combination, arcdps sep27 AND sep28 release tested
without arcdps everything works fine

**Comment by Friendly0Fire** [comment]
Yeah, we're looking into it, it appears to be a problem with arcdps in certain configurations.
